terraform/az-monitoring/module/main.tf

```diff
@@ -24,7 +24,8 @@ resource "aws_security_group" "access_sg" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
+    // https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-set-up.html#ec2-instance-connect-setup-security-group
+    cidr_blocks = ["18.202.216.48/29", "3.8.37.24/29"]
```

Comment:
@paroxp what are theses ips referencing??

Reply:
Essentially, it's the CIDR from both London and Ireland AWS regions, claiming only these IPs can use port 22.

More specifically, the "EC2 Instance Connect" via web console.

The whole process is described in the link in the comment.